OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -19,6 +19,9 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Renders a decorative animation on units and buildings.")]
 	public class WithIdleOverlayInfo : UpgradableTraitInfo, IRenderActorPreviewSpritesInfo, Requires<RenderSpritesInfo>, Requires<IBodyOrientationInfo>
 	{
+		[Desc("Animation to play when the actor is created.")]
+		[SequenceReference] public readonly string StartSequence = null;
```

Comment:
This doesn't do anything on its own, look at the implementation in `WithSpriteBody` to see how this is played before the looping animation, and then change line 73 accordingly.